src/test/java/com/edu/MyNumberTest.java

```diff
@@ -0,0 +1,30 @@
+package com.edu;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MyNumberTest {
+
+    MyNumber myNumber = new MyNumber();
+    int validationCnt = 0;
+
+    @ParameterizedTest
+    @ValueSource(strings = {"123", "486", "387", "267", "954", "527"})
+    void myNumTrueValidation(String myNum) {
+
+        validationCnt = myNumber.validation(myNum, validationCnt);
+        assertThat(validationCnt <= 0).isTrue();
```

Comment:
```suggestion
        assertThat(validationCnt == 0).isTrue();
```
의도하는 코드는 값이 유요한 경우 validationCnt가 0이 나와야 함으로  ``< 0`` 조건을 배제하는 것이 더 정확한 테스트라고 볼 수 있습니다.